builder/dispatchers.go

```diff
@@ -253,10 +253,19 @@ func run(b *Builder, args []string, attributes map[string]bool, original string)
 	c.Mount()
 	defer c.Unmount()
 
+	// Unset the volumes
+	volumeConfig := c.Config.Volumes
+	c.Config.Volumes = nil
+	volumes := c.Volumes
+	c.Volumes = nil
+
 	err = b.run(c)
```

Comment:
I was thinking about this... technically I think we'd want the real information about the container that actually ran.
wdyt?